src/test/java/org/dcsa/api_validator/bkg/v1/GetBookingRequestsTest.java

```diff
@@ -0,0 +1,262 @@
+            OffsetDateTime dateTime2 = OffsetDateTime.parse(submissionDateTimesSorted.get(i), DateTimeFormatter.ISO_OFFSET_DATE_TIME);
+            Assert.assertTrue(dateTime1.compareTo(dateTime2) >= 0);
+        }
+    }
+
+    // Test sorting:ASC (descending) and if order is respected. for BookingRequestDateTime
+    @Test
+    public void testSortAscBookingRequestDateTimeQueryParam() {
+
+        List<String> bookingRequestDateTimesSorted = getListOfAnyAttribute("bookingRequestDateTime", "sort", "bookingRequestDateTime:ASC");
+        assert (!bookingRequestDateTimesSorted.isEmpty());
+        int n = bookingRequestDateTimesSorted.size();
+
+        for (int i = 1; i < n; i++) {
+            OffsetDateTime dateTime1 = OffsetDateTime.parse(bookingRequestDateTimesSorted.get(i - 1), DateTimeFormatter.ISO_OFFSET_DATE_TIME);
+            OffsetDateTime dateTime2 = OffsetDateTime.parse(bookingRequestDateTimesSorted.get(i), DateTimeFormatter.ISO_OFFSET_DATE_TIME);
+            Assert.assertTrue(dateTime1.compareTo(dateTime2) <= 0);
+        }
+    }
+
+    // Test sorting:DESC (descending) and if order is respected. for BookingRequestDateTime
+    @Test
+    public void testSortDescBookingRequestDateTimeQueryParam() {
+
+        List<String> bookingRequestDateTimesSorted = getListOfAnyAttribute("bookingRequestDateTime", "sort", "bookingRequestDateTime:DESC");
+        assert (!bookingRequestDateTimesSorted.isEmpty());
+        int n = bookingRequestDateTimesSorted.size();
+
+        for (int i = 1; i < n; i++) {
+            OffsetDateTime dateTime1 = OffsetDateTime.parse(bookingRequestDateTimesSorted.get(i - 1), DateTimeFormatter.ISO_OFFSET_DATE_TIME);
+            OffsetDateTime dateTime2 = OffsetDateTime.parse(bookingRequestDateTimesSorted.get(i), DateTimeFormatter.ISO_OFFSET_DATE_TIME);
+            Assert.assertTrue(dateTime1.compareTo(dateTime2) >= 0);
+        }
+    }
+
+    @Test
+    public void testLimitQueryParam() {
+
+        List<Integer> limits = Arrays.asList(1, 2, 5, 6);
+
+        limits.forEach(limit -> {
+            given().
+                    auth().
+                    oauth2(Configuration.accessToken).
+                    queryParam("limit", limit).
+                    get(Configuration.ROOT_URI + "/booking-summaries").
+                    then().
+                    assertThat().
+                    statusCode(200).
+                    body("size()", equalTo(limit)).
+                    body(matchesJsonSchemaInClasspath("bkg.v1/BookingRequestsSchema.json").using(jsonSchemaFactory));
+        });
+    }
+
+    // Test limit, fails as formatting is wrong.
+    @Test
+    public void testLimitFalseFormat() {
+        given().
+                auth().
+                oauth2(Configuration.accessToken).
+                // Specification -> minimum: 1
+                        queryParam("limit", "0").
+                get(Configuration.ROOT_URI + "/events").
```

Comment:
Should the URI have been `/booking-summaries` and not `/events`?

Reply:
I must have overlooked this. Thanks